sw/device/silicon_creator/lib/cert/cbor.h

```diff
@@ -6,100 +6,52 @@
 #define OPENTITAN_SW_DEVICE_SILICON_CREATOR_LIB_CERT_CBOR_H_
 
 #include "include/dice/cbor_writer.h"
-#include "sw/device/lib/runtime/log.h"
 #include "sw/device/silicon_creator/lib/error.h"
 
-#define CBOR_CHECK_OVERFLOWED_AND_RETURN(p) \
-  do {                                      \
-    if (CborOutOverflowed(p)) {             \
-      LOG_ERROR("CborOutOverflowed!!");     \
-      return kErrorCertInvalidSize;         \
-    }                                       \
-    return kErrorOk;                        \
-  } while (0)
-
 // Wrappers for each CBOR type and CBOR handle initialization
-static inline rom_error_t cbor_write_out_init(struct CborOut *p, void *buf,
-                                              const size_t buf_size) {
-  CborOutInit(buf, buf_size, p);
-  CBOR_CHECK_OVERFLOWED_AND_RETURN(p);
-}
-
-static inline rom_error_t cbor_map_init(struct CborOut *p,
-                                        const size_t num_pairs) {
-  CborWriteMap(num_pairs, p);
-  CBOR_CHECK_OVERFLOWED_AND_RETURN(p);
-}
-
-static inline rom_error_t cbor_array_init(struct CborOut *p,
-                                          const size_t num_elements) {
-  CborWriteArray(num_elements, p);
-  CBOR_CHECK_OVERFLOWED_AND_RETURN(p);
-}
-
-static inline rom_error_t cbor_write_string(struct CborOut *p,
-                                            const char *str) {
-  CborWriteTstr(str, p);
-  CBOR_CHECK_OVERFLOWED_AND_RETURN(p);
-}
-
-static inline rom_error_t cbor_write_bytes(struct CborOut *p,
-                                           const uint8_t *data,
-                                           const size_t data_size) {
-  CborWriteBstr(data_size, data, p);
-  CBOR_CHECK_OVERFLOWED_AND_RETURN(p);
-}
+rom_error_t cbor_write_out_init(struct CborOut *p, void *buf,
```

Comment:
all functions prototypes in the header file should have doxygen style comments: https://cs.opensource.google/opentitan/opentitan/+/master:sw/device/lib/testing/i2c_testutils.h;l=158